docker/google-cloud-securitycenter/Dockerfile

```
@@ -0,0 +1,8 @@
+
+FROM demisto/python3:3.8.6.14516
+
+COPY requirements.txt .
+
+RUN apk --update add --no-cache --virtual .build-dependencies python3-dev build-base wget git \
```

Need to add `linux-headers` so the install compiles.
```suggestion
RUN apk --update add --no-cache --virtual .build-dependencies python3-dev build-base wget git linux-headers \
```

---

Seems to be same as reported here: https://github.com/grpc/grpc/issues/21918